recipes/video-playback.md

```diff
@@ -70,7 +70,7 @@ mpv:
 ### Run
 
 ```bash
-mpv --vo=gpu --gpu-context=drm --hwdec=rkmpp video.mkv
+mpv --vo=gpu --gpu-context=drm --hwdec=rkmpp --drm-mode=21 video.mkv
```

Comment:
Can we add `--drm-mode=21` to maybe new section `Quirks`?

It can help in some cases.